src/init.py

```diff
@@ -0,0 +1,128 @@
+import glob
+import os
+import pathlib
+import shutil
+import subprocess
+from itertools import product
+from build import build
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+import logging
+import sys
+
+
+def getvar(var: str) -> str:
+    val = os.getenv(var)
+    if val == "" or val is None:
+        raise ValueError('Environment variable "%s" has an invalid value.' % var)
+    return val
+
+
+def make_key(key_path: str, key_subj: str) -> None:
+    subprocess.run(
+        ["/root/make_key", key_path, key_subj],
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+        check=True,
+        input="\n".encode(),
+    )
+
+
+def init() -> None:
+    # Copy the user scripts
+    root_scripts = "/root/user_scripts"
+    user_scripts = getvar("USERSCRIPTS_DIR")
+    shutil.copytree(user_scripts, root_scripts)
+
+    # Delete non-root files
+    to_delete = []
+    for filename in glob.iglob(os.path.join(root_scripts, "**/**"), recursive=True):
+        if os.path.isdir(filename):
+            continue
+
+        # Check if not owned by root
+        f = pathlib.Path(filename)
+        if f.owner != "root":
+            logging.warning("File not owned by root. Removing %s", filename)
+            to_delete.append(filename)
+            continue
+
+        # Check if non-root can write (group or other)
+        perm = oct(os.stat(filename).st_mode)
+        group_write = perm[-2] > "4"
+        other_write = perm[-1] > "4"
+        if group_write or other_write:
+            logging.warning("File writable by non root users. Removing %s", filename)
+            to_delete.append(filename)
```

Comment:
```suggestion
    for path in Path(root_scripts).rglob("*"):
        if path.is_dir():
            continue

        # Check if not owned by root
        if path.owner != "root":
            logging.warning("File not owned by root. Removing %s", path.resolve())
            to_delete.append(path)
            continue

        # Check if non-root can write (group or other)
        perm = oct(path.stat().st_mode)
        group_write = perm[-2] > "4"
        other_write = perm[-1] > "4"
        if group_write or other_write:
            logging.warning("File writable by non root users. Removing %s", path.resolve())
            to_delete.append(path)
```

Reply:
This has been implemented in latest version